server.js

```diff
@@ -21,8 +21,8 @@
 const http = require('http');
 const { MongoClient } = require('mongodb');

```

Comment:
Consider using a more descriptive name for the HOSTNAME constant, such as SERVER_HOST, to improve code readability.